### DISCUSSION TITLE
HELP: How to use System.minimize()

### DISCUSSION THREAD

**Discussion by ZhouXY-PKU** [deepmodeling/dpdata#519] [Q&A] [answered]
Hello everyone, 
I have found there is System.minimize(), but there seems no example/manual of this attribute. I have also failed to open https://github.com/deepmodeling/dpdata/blob/master/docs/minimizers.rst, So are there any example of this attribute?
Thank you!

**Comment by wanghan-iapcm** [comment, marked as answer]
https://docs.deepmodeling.com/projects/dpdata/en/master/api/dpdata.html#dpdata.MultiSystems.minimize

@njzjz the format of the example is problematic.

**Reply by njzjz** [reply]
I fix it in #523.

**Reply by ZhouXY-PKU** [reply]
Thanks, but the example seems wrong? #524
